azure-resources/Sql/servers/recommendations.yaml

```diff
@@ -46,9 +46,9 @@
   recommendationResourceType: Microsoft.Sql/servers
   recommendationMetadataState: Active
   longDescription: |
-    By default, the premium availability model clusters nodes in one datacenter.
+    By default, when you provision an Azure SQL Database with the premium tier, multiple copies of the database are created within the same azure region. To support geo redundancy, Databases can be provisioned (or existing databases can be modified) as Zone Redundant, which locates the copies in different Azure Availability Zones. This will ensure database availability in the event of a regional outage.
```

Comment:
Please re-word to bring the longDescription under the 300 character limit, including spaces.